samples/Quickstart/src/Filters/QuickstartFilter.cs

```diff
@@ -9,55 +19,89 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json.Linq;
-using System;
-
-using System.Threading.Tasks;
 
 namespace Quickstart.Filters
 {
     public class QuickstartFilter : IInputFilter
     {
-        public QuickstartFilter(IOptions<QuickstartOptions> options, IAuthenticator authenticator, TelemetryClient telemetryClient = null, ILogger<QuickstartFilter> logger = null)
+        public QuickstartFilter(IHttpCustomHeaderCollection customHeader, TelemetryClient telemetryClient = null, ILogger<QuickstartFilter> logger = null)
         {
-            id = Guid.NewGuid().ToString();
-            fhirServerUrl = options.Value.FhirServerUrl;
-            retryDelaySeconds = options.Value.RetryDelaySeconds;
-            maxRetryAttempts = options.Value.MaxRetryAttempts;
-            status = options.Value.ExecutionStatusType;
-            this.authenticator = authenticator;
-            this.telemetryClient = telemetryClient;
-            this.logger = logger;
+            _id = Guid.NewGuid().ToString();
+            _telemetryClient = telemetryClient;
+            _logger = logger;
+            _customHeader = customHeader;
```

Comment:
Please delete the _customHeader variable and any lines referencing it in the filter...RestBinding will do this work.

Reply:
Removed as suggested. please verify.